CRM/Chreports/Reports/ReportConfiguration.php

```diff
@@ -351,11 +351,20 @@ public function buildJsonConfigSettings(): void {
                 if($vOrder['column'] == '-') {
                     continue;
                 }
-                $config['order_bys'][$vOrder['column']] = $vOrder;
-                if(isset($vOrder['section'])) {
-                    //Rename Section -> Header instead
-                    $config['order_bys'][$vOrder['column']]['header'] = $vOrder['section'];
-                    unset($config['order_bys'][$vOrder['column']]['section']);
+                // Check for custom field
+                // if it's a custom field, get the key value from mapping.json
+                $orderByKey = $vOrder['column'];
+                if(strpos($orderByKey, 'custom_') === 0) {
+                    $orderByKey = $this->getCustomFieldFromMappingJson($orderByKey);
+                    unset($vOrder['column']);
+                }
+                if(!empty($orderByKey)) {
+                    $config['order_bys'][$orderByKey] = $vOrder;
+                    if(isset($vOrder['section'])) {
+                        //Rename Section -> Header instead
+                        $config['order_bys'][$vOrder['column']]['header'] = $vOrder['section'];
+                        unset($config['order_bys'][$vOrder['column']]['section']);
+                    }
```

Comment:
Adding log if can't map field.

```suggestion
                    }
            } else {
                    watchdog("reporting", "Failed to map field: ".$orderByKey . " -> ".$config['title'] . "", [], WATCHDOG_DEBUG);            
```